ui/StatusQ/src/StatusQ/Controls/StatusTagItem.qml

```diff
@@ -88,7 +90,8 @@ Control {
         }
         StatusBaseText {
             color: Theme.palette.indirectColor1
-            font.pixelSize: 15
+            font.pixelSize: root.font.pixelSize
+            font.bold: root.font.bold
```

Comment:
Nope, better just do:
```qml
font: root.font
```

Reply:
That way you can extend this to anything in the future

Reply:
Ok! Good idea!